tests/llmcompressor/transformers/obcq/test_consecutive_runs.py

```diff
@@ -137,10 +138,15 @@ class TestConsecutiveRunsGPU(TestConsecutiveRuns):
     def setUp(self):
         from transformers import AutoModelForCausalLM
 
+        kwargs = {}
+        # if optimized self.model is passed, then must be using Linear Modules
+        if is_model_ct_quantized_from_path(self.model):
```

Comment:
This is fine but likely overkill as the models for this test won’t be quantized to start

Reply:
Ok I can add an assertion instead. Its good to start from dense i agree